### ISSUE TITLE
cinnamon-screensaver makes cinnamon hang after upgrading to ulyssa

### ISSUE THREAD

**Issue by jrversteegh** [linuxmint/cinnamon#9860]
* Cinnamon version 4.8.6, cinnamon-screensaver 4.8.1
 * Distribution - (Mint 20.1, x86_64, etc...)
 * Graphics hardware: intel + amd vega 20
The setup is with two graphics adapters and two monitors. Running `cinnamon-screensaver --interactive-debug` shows:
```
Cinnamon Screensaver support not found in current theme - adding some...
Trying to connect to logind...
Starting screensaver...
Successfully using logind
Cinnamon Screensaver support not found in current theme - adding some...
couldn't grab keyboard
couldn't grab keyboard
couldn't grab keyboard
couldn't grab keyboard
....
```

**Comment by mtwebster** [comment]
Is this only when manually locking? Or idle locking?

Also can you run it with --debug instead of --interactive-debug

That log there is also strange in that the "support not found in current theme..." message - you should only receive that once.

Make sure there's only one process running (though it really shouldn't be possible to have more than one anyhow):
```
ps -A | grep cinnamon-screen
```


Thanks

**Comment by jrversteegh** [comment]
OK, I got a 404 on the first submit, but apparently the issue was created, sorry about that.
It looks like this issue has been resolved after `apt install --reinstall cinnamon-screensaver`.Before that I removed `~/.config/monitors.xml` and now `~/.config/cinnamon-monitors.xml` has been created. There was a message about that in `.xsession-errors` as well as `(cinnamon-screensaver ): Gdk-CRITICAL **:  gdk_device_get_source: assertion 'GDK_IS_DEVICE (device)' failed` , but not entirely sure if it was related.
